packages/introspect/src/e2e.test.ts

```diff
@@ -22,7 +22,9 @@ describe('e2e', () => {
                     parser: 'typescript',
                     ...prettierOptions,
                 }),
-            SECRET_INTERNALS_DO_NOT_USE_queriesImportLocation: '../query',
+            SECRET_INTERNALS_DO_NOT_USE_queryBuilderImportLocation: '../query',
```

Comment:
You don't need two secret params. You can just have a single one that points to the "index" file and from there export everything you need.